src/scancode/cli.py

```diff
@@ -383,6 +384,8 @@ def scan_event(item):
                 if item:
                     _scan_success, _scanned_path = item
                     _progress_line = verbose and _scanned_path or fileutils.file_name(_scanned_path)
+                    if not verbose:
+                    	_progress_line = shorten_filename(_progress_line)
```

Comment:
@pombredanne The logic here is such that `_progress_line` will contain the file path only if `verbose` is set to True. In that case `shorten_filename()` will not be called at all. So argument passed to  `shorten_filename()` will always be filename only and not the file path. So why are we testing `shorten_filename()` with a file path?